docs/releases/1.11.7.txt

```diff
@@ -11,5 +11,9 @@ Bugfixes
 
 * Prevented ``cache.get_or_set()`` from caching ``None`` if the ``default``
   argument is a callable that returns ``None`` (:ticket:`28601`).
+* ...
+
+* Fixed the Basque ``DATE_FORMAT`` string (:ticket:`28710`).
+* ...
```

Comment:
it looks like you've merged too much stuff into this branch. you probably want to rebase on top of master, rather than merge. use `git rebase --interactive` to select only your commits when you do so, to avoid pulling in these other changes.